examples/anime/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Bouncy Ball - Comparing Web Animation Techniques - anime.js</title>
+  <link rel="stylesheet" href="../shared-styles.css">
+  <style>
+    #ball {
+      display: block;
+      width: 50px;
+      height: 50px;
+      border-radius: 50%;
+      background-color: #ae27e7;
```

Comment:
Nice color! Can we also add it to the bottom of [this list of link colors in the CSS](https://github.com/sparkbox/bouncy-ball/blob/65673f33b75d49fbf415f5ce1d74b2e4c0266cae/site/css/styles.css#L140-L149)? This is the button text color for when the demo is selected. Currently, it just show black text on a dark background 😞 :

![image](https://cloud.githubusercontent.com/assets/1256329/19008128/1a7b2e68-8737-11e6-808a-44bf865ee2e2.png)